server/app.ts

```diff
@@ -41,6 +42,7 @@ export default function createApp(userService: UserService): express.Application
   app.use('/select-location', selectLocationRoutes(standardRouter(userService)))
   app.use('/incentive-summary/:locationPrefix', incentivesTableRoutes(standardRouter(userService)))
   app.use('/prisoner-images/:imageId.jpeg', prisonerImagesRoutes(imageRouter()))
+  app.use('/analytics', analyticsRouter(standardRouter(userService)))
```

Comment:
I think this route should be added only if the feature is enabled.

I know the card would be hidden but potentially you could get there directly. In this case it probably doesn't matter much because there wouldn't be any harm, but I thought I'd mention.

Reply:
instead, i'll make it 404. otherwise tests won't work